changes.go

```diff
@@ -51,8 +51,7 @@ func (c *pollChangeManager) close() <-chan bool {
 	}
 	// not launched
 	if atomic.LoadInt32(c.isLaunched) == int32(0) {
-		log.Debug("pollChangeManager: close() called when pollChangeWithBackoff unlaunched! Will wait until pollChangeWithBackoff is launched and then kill it and tokenManager!")
-		log.Warn("Attempt to close unstarted change manager")
+		log.Warn("pollChangeManager: close() called when pollChangeWithBackoff unlaunched! Will wait until pollChangeWithBackoff is launched and then kill it and tokenManager!")
```

Comment:
Not exactly it's allowed, and can be handled properly, no panic needed.
The code can handle this properly, whenever you close it , it will be closed gracefully.